source/dpp/translation/variable.d

```diff
@@ -53,11 +53,14 @@ string[] translateVariable(in from!"clang".Cursor cursor,
     if(constexpr)
         ret ~= translateConstexpr(spelling, cursor, context);
     else {
-
+        const typeSpelling = translateType(cursor.type, context, No.translatingFunction);
+        // In C it is possible to have an extern void variable
+        const typeNotVoid = (typeSpelling == "void" || typeSpelling == "const(void)")
```

Comment:
Why would you translate first then inspect the results? I'd check if `cursor.type.kind == Type.Kind.Void`.